src/enzo/hydro_rk/EOS.h

```diff
@@ -143,6 +144,38 @@ inline void EOS(float &p, float &rho, float &e, float &h, float &cs, float &dpdr
 
   }
 
+  /* Only works in mode 1 and only updates internal energy */
```

Comment:
Need to add to the EOS documentation to specify that this only works in mode 1 (or add an assertion in the code)

Reply:
Added an exception